source/_docs/guides/wordpress-commandline.md

```diff
@@ -55,59 +56,45 @@ Creating a site is a function of the Pantheon API, not WP-CLI; however, Terminus
 
-The format for creating a site with a single command is:
-
-```nohighlight
-site:create [--org [ORG]] [--] <site_name> <label> <upstream_id>
-```
+Optionally you can also include **Organization ID (or Name)** at the end of the `site:create` command. This optional parameter lets you associate the created site with a Pantheon organization. If you do not work with a team of other developers, you might not belong to any organizations. To see the list of organizations associated with your user account, run `terminus org:list`
 
 For my test site, I used the following:  
 **Upstream** = WordPress
 **Site Name** = cli-test  
-**Label** = Command Line Test
+**Label** = Terminus CLI Create
 
-```nohighlight
-$ terminus site:create YOUR-ORG-ID cli-test "Terminus CLI Create" e8fe8550-1ab9-4964-8838-2b9abdccf4bf
-```
 **Note:** Copying this command will fail, because the site name is now taken. Choose a different name for your test.
 
 Just like when you create a site from your Dashboard, this will only take a few minutes. You will see a status bar as terminus creates your new WordPress installation. Once complete, you will be notified that you site is ready to go.
 
-From Terminus, you can get to your Site Dashboard with `$ terminus dashboard <site>.<env>`
-
-## Clone the Codebase and Dotenv
-Run the following command within backticks to [create a local repository of your site's codebase](/docs/git):
-
-```
-terminus dashboard:view <site>.<env>
-```
+From Terminus, you can get to your Site Dashboard with `$ terminus dashboard <site>.<env>`. You might also want to [clone your site's repository locally at this point](/docs/git)
 
 ## Install WordPress
 
 Now that WordPress code is there, it's time for step five of the "[Famous 5-minute  Install](http://codex.wordpress.org/Installing_WordPress#Famous_5-Minute_Install)". Steps 1-4 were completed for you by Pantheon, and you don't need anything but the command line to finish. There's even a `wp-config.php` already created and ready to use.
 
-All you need to do now is populate the database and your site will be ready to use. Using Terminus and WP-CLI running on the server, use the `wp core install` command. For this to work, it's necessary that you understand the [wp-cli core install](http://wp-cli.org/commands/core/install/) command:
+All you need to do now is populate the database and your site will be ready to use. Using Terminus and WP-CLI running on the server, use the `wp core install` command. For this to work, it's necessary that you understand the [wp-cli core install](http://wp-cli.org/commands/core/install/) command. You will need to update a few parameters in this command before running it:`--title`, `--admin_user`, and `--admin_email`.
 
 ```nohighlight
-terminus wp -- core install --url=http://dev-cli-test.pantheion.io --title="WP-CLI-Test" --admin_user=admin --admin_password=something_incredibly_secure --admin_email=your@emailaddress.tld
```

Comment:
This is the command that is missing the site.env piece. I will remove the username and password. :)